pkg/conversion/mapi2capi/aws.go

```diff
@@ -514,6 +510,18 @@ func convertKMSKeyToCAPI(kmsKey mapiv1.AWSResourceReference) string {
 	return ""
 }
 
+func convertEBSVolumeSizeToCAPI(bdm *mapiv1.BlockDeviceMappingSpec) int64 {
+	// If bdm.EBS.VolumeSize is not nil, return the VolumeSize
+	if bdm.EBS.VolumeSize != nil {
+		return *bdm.EBS.VolumeSize
+	}
+
+	// Otherwise, return the default VolumeSize (in GB)
+	defaultVolumeSize := int64(120)
```

Comment:
I'm not 100% on this being a reasonable default value right now - need to do some more research on this